### PR TITLE
Add goccm

### PR DESCRIPTION
**Pull request by zenthangplus**
- github.com repo: https://github.com/zenthangplus/goccm
- pkg.go.dev: https://pkg.go.dev/github.com/zenthangplus/goccm?tab=doc
- goreportcard.com: https://goreportcard.com/report/github.com/zenthangplus/goccm
- coverage service link: [codecov](https://codecov.io/gh/zenthangplus/goccm)

Very good coverage

- [X] I have added my package in alphabetical order.
- [X] I have an appropriate description with correct grammar.
- [X] I know that this package was not listed before.
- [X] I have added pkg.go.dev link to the repo and to my pull request.
- [X] I have added coverage service link to the repo and to my pull request.
- [X] I have added goreportcard link to the repo and to my pull request.
- [X] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

### CODE REVIEW

**Comment by avelino** [comment]
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body links content check:
- godoc.org or pkg.go.dev: True
- goreportcard.com: True
- coverage: True

File: README.md [hunk 1]
```diff
@@ -1878,7 +1878,8 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [go-sitemap-generator](https://github.com/ikeikeikeike/go-sitemap-generator) - XML Sitemap generator written in Go.
 * [go-trigger](https://github.com/sadlil/go-trigger) - Go-lang global event triggerer, Register Events with an id and trigger the event from anywhere from your project.
 * [goback](https://github.com/carlescere/goback) - Go simple exponential backoff package.
-* [godaemon](https://github.com/VividCortex/godaemon) - Utility to write daemons.
+* [goccm](https://github.com/zenthangplus/goccm) - Go Concurrency Manager package limits the number of goroutines that allowed to run concurrently.
+* [godaemon](https://github.com/VividCortex/godaemon) - Utility to wrisimple exponential backoff package.te daemons.
```
Comment (sanketplus) [inline README.md:1882]: `write simple` typo??

Comment (zenthangplus) [reply README.md]: Thank you. I've updated

**Comment by zenthangplus** [comment]
@sanketplus i've resolved. Could you merge? Thank

**Comment by sanketplus** [comment]
Apart from the `plus` postfix, the other thing we got in common is lack of merge privilage 😸 @zenthangplus

**Comment by zenthangplus** [comment]
> You should put this repo into [Goroutines](https://github.com/avelino/awesome-go#goroutines)

Thank you. I've updated

**Pull request merged**